programmers2/신규-아이디-추천.js

```diff
@@ -0,0 +1,14 @@
+function solution(new_id) {
+    new_id = new_id
+    .toLowerCase()
+    .replace(/[^\w-.)]/g, '')
+    .replace(/\.{1,}/g, '.')
```

Comment:
1을 적을시 1개 이상인것도 전부 . 하나로 바꿔주는것인데
2를 적어야 주어진대로 2개이상일때 . 하나로 바뀌는 작업이 이루어집니다.
둘다 . 한개로 나오는 같은 모습이지만 주어진 조건에 맞추려면 2가 적합해 보입니다 :D
```suggestion
    .replace(/\.{2,}/g, '.')
```